.github/workflows/gardening.yml

```diff
@@ -1,7 +1,7 @@
 name: Gardening
 on:
   pull_request_target: # When a PR is opened, edited, updated, closed, or a label is added.
-    types: [opened, reopened, synchronize, edited, labeled, closed]
+    types: [opened, reopened, synchronize, edited, labeled, closed, converted_to_draft]
```

Comment:
If you add a new type here, I believe you would also have to also use it in the action, here:
https://github.com/Automattic/jetpack/blob/ea6e214fd0fae66e14e27f55fb8fa4997a679e1d/projects/github-actions/repo-gardening/src/index.js#L30

Reply:
> Should we also remove other existing Status labels as well as adding the In Progress label?

I think that would make more sense, otherwise we can run into the case of having conflicting status labels. I am, however, reluctant to add logic to remove labels added by authors (except the clean-up of status labels when merged). So I'll remove the `converted_to_draft` type, and only add the in progress when a draft PR is opened. This was the initial intention.

Reply:
Removed with [8567262](https://github.com/Automattic/jetpack/pull/25130/commits/8567262196e98968709040d52d3e2de484820b0a)